src/Makefile

```diff
@@ -121,10 +125,14 @@ GHERKIN_CLI_OBJS= \
 	$(MKDIR_CMD) $(dir $@)
 	$(AR) $(AR_FLAGS) $@ $(UTILITIES_OBJS) $(PARSER_OBJS) $(AST_OBJS) $(COMPILER_OBJS) $(PICKLES_OBJS) $(EVENT_OBJS)
 
+../libs/libgherkin.so: $(UTILITIES_OBJS) $(PARSER_OBJS) $(AST_OBJS) $(COMPILER_OBJS) $(PICKLES_OBJS) $(EVENT_OBJS)
```

Comment:
Unless declared ".PHONY", make rules shall create their target, so the make target should be "../libs/libgherkin.so".

Reply:
I do not understand what you mean (again, my make experience is not great) so I am unable to comply.  If you can make the change so I can see what you mean, that would be very helpful.

Reply:
["A phony target is one that is not really the name of a file"](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html#Phony-Targets). "`../libs/libgherkin.so`" is "not really the name of a file" (the name of the file is "`../libs/libgherkin.so.$(VERSION)`". Since the basic principle of make is that the target of a rule is a file (unless there is a very compelling reason otherwise - for instance for rules with the target "all"), "`../libs/libgherkin.so.$(VERSION)`" is a much better target for this rule. It also enable the use of the [automatic variable `$@`](https://www.gnu.org/software/make/manual/html_node/Automatic-Variables.html)